swiftuploader/upload.py

```diff
@@ -3,36 +3,44 @@
 import logging
 import optparse
 import os
+import os_client_config
 import sys
 import re
 import time
 import hashlib
+import yaml
 
 from openstack import connection
-from openstack import profile
-from openstack import utils
+
+
+PWD_COVER = 'XXXXXX'
```

Comment:
Do you mean that this is just a replaced string?  If so this should be "<password>" or similar

Reply:
Yes, it just used to put in clouds.yaml as a place holder. Will change to ""